src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -687,6 +698,19 @@ ReactDOMComponent.Mixin = {
   },
 
   /**
+   * Setup this component to trap non-bubbling events locally
```

Comment:
I'm not sure if this description suffices but I didn't know how else to describe what it does!